src/main/java/de/siphalor/nmuk/api/NMUKAlternatives.java

```diff
@@ -36,47 +37,72 @@
 	 *
 	 * @param base The base keybinding to create an alternative for
 	 * @param code The keycode to use as default for the alternative
+	 * @return the new created keybinding
 	 */
-	public static void create(KeyBinding base, int code) {
-		create(base, InputUtil.Type.KEYSYM, code);
+	public static KeyBinding create(KeyBinding base, int code) {
```

Comment:
I generally agree to this, but I have two notes:

1. This change breaks byte code compatibility which would result in a major version bump. I'm not sure if I want this here — maybe this should get a different name for now.
2. It should probably be generified if it returns its argument, so it works better with custom classes.

Reply:
To 1.: Should I leave the old method signature and make it deprecated and give the new method a different name? Or should I not deprecate the old one?
To 2.: 2 of the 3 method named "create" return a new KeyBinding not the one given as argument. Only the method with `KeyBinding alternative` as argument return the given KeyBinding. So I only generify that one